blazor/treeview/animation.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: post
+title: Animation in Blazor TreeView Component | Syncfusion
+description: Checkout and learn here all about Animation in Syncfusion Blazor TreeView component and much more.
+platform: Blazor
+control: TreeView
+documentation: ug
+---
+
+# Animation in Blazor TreeView Component

```

Comment:
What will I do if I specify the effect alone in animation settings?
How to disable animation?

Reply:
1. What will I do if I specify the effect alone in animation settings?

- If we use only the effect property in Animation settings, the used effect will apply.

2. How to disable animation?

- we can disable animation by setting the Duration value as 0.

3. animation topic can be grouped under 'expand and collapse'. (have any topic already for it?)

- Done, sir.

4. Is TreeView animation works while programmatically expand/collapse a node?

- We will ensure this case, sir.